include/ck/tensor_operation/operator_transform/transform_contraction_to_gemm.hpp

```diff
@@ -0,0 +1,285 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2018-2022, Advanced Micro Devices, Inc. All rights reserved.
+
+#pragma once
+
+#include "ck/utility/common_header.hpp"
+#include "ck/tensor_operation/gpu/device/matrix_padder.hpp"
+#include "ck/tensor_operation/gpu/device/gemm_specialization.hpp"
+
+namespace ck {
+namespace tensor_operation {
+
+// assume C[G0, G1, ..., M0, M1, M2, ..., N0, N1, N2...]
+template <index_t MPerBlock,
+          index_t NPerBlock,
```

Comment:
Does ```MakeGridDescriptorPair``` really use ```MPerBlock``` and ```NPerBlock```

Reply:
No it doesn't. Actually an artifact from a few refactors before. Nice catch

Reply:
MNPerBlock is redundant and removed in latest commits